src/users/api/permissionDto.ts

```diff
@@ -0,0 +1,6 @@
+import { ApiProperty } from '@nestjs/swagger';
+
+export class permissionDto {
```

Comment:
Use `PascalCase`  for the name of classes

```suggestion
export class PermissionDto {
```